api/v1alpha1/etcd_types.go

```diff
@@ -141,6 +141,9 @@ type BackupSpec struct {
 	// EnableProfiling defines if profiling should be enabled for the etcd-backup-restore-sidecar
 	// +optional
 	EnableProfiling *bool `json:"enableProfiling,omitempty"`
+	// CompactionSchedule defines the cron standard for compacting the snapstore
+	// +optional
+	CompactionSchedule *string `json:"compactionSchedule,omitempty"`
```

Comment:
Should we call this `backupCompactionSchedule` just to avoid confusing with ETCD compaction?

Reply:
Done